### ISSUE TITLE
4.5.2 hangs HHVM with 100% cpu usage

### ISSUE THREAD

**Issue by r3wt** [phpmyadmin/phpmyadmin#11707]
the issue is intermittent and after a while it goes away entirely.

things that trigger it:

- login, every single time
- viewing a table, intermittent
- editing a column, intermittent
- changing user password via usermanager, intermittent

https://github.com/facebook/hhvm/issues/6429

I have been able to reproduce the issue on every single server i've tested.

**Comment by lem9** [comment]
Thanks. If by "latest" you mean 4.5.1 as in your Facebook report, did you try 4.5.2?

**Comment by r3wt** [comment]
issue is still present in 4.5.2 yes

**Comment by lem9** [comment]
So it must be HHVM that incorrectly executes PHP code?

**Comment by sschakraborty** [comment]
I faced exactly the same problem. Whenever I logged into phpMyAdmin and went through a few clicks, everything hung and CPU usage rocketed to 100%.

I found out that the problem was probably with sessions as whenever I cleared Chrome or Firefox session cache and tried to reopen phpMyAdmin again, it responded. However HHVM was still peaked at 100% CPU usage.

I enabled JIT compilation from /etc/hhvm/config.hdf and it solved my problem. I request you to try this and tell me if it worked for you too.

P.S.: To enable JIT on HHVM refer to http://hhvm.com/blog/4061/go-faster
